crates/js-sys/src/lib.rs

```diff
@@ -3392,6 +3392,13 @@ extern "C" {
     #[wasm_bindgen(method, js_name = toLocaleTimeString)]
     pub fn to_locale_time_string(this: &Date, locale: &str) -> JsString;
 
+    #[wasm_bindgen(method, js_name = toLocaleTimeString)]
+    pub fn to_locale_time_string_with_options(
+        this: &Date,
+        locale: &str,
+        options: &JsValue,
```

Comment:
```suggestion
        options: Option<&Object>,
```
Is probably more accurate, again, deviating more from the `to_local_time()`. So lets just leave it for the next breaking change.